src/editor/Core/Buffer.re

```diff
@@ -16,12 +16,11 @@ let ofLines = (lines: array(string)) => {
   lines,
 };
 
-let show = (b: t) => {
+let show = (b: t) =>
```

Comment:
I'm glad we're integrating `ppx_deriving.show`, we can get rid of all these random, manual show methods 😍